x-pack/test_serverless/functional/test_suites/apps/lens/group1/config.ts

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Hi. We don't want to introduce additional sub-directories in `test_suites`. Depending on where you want to run your tests, you should either nest them in one of the projects (`test_suites/observability`, `test_suites/search`, `test_suites/security`) or in `test_suites/common` if the functionality is present (and should be tested) in more than one project (see also the serverless test [README](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/README.md#common-tests)).

When nesting under one of the existing test suite directories, you also don't have to repeat the `createTestConfig` call but could just load the parent config and override the bits you need, e.g. when moving to  `test_suites/search/lens`, the config could look like this:
```
import { FtrConfigProviderContext } from '@kbn/test';

export default async ({ readConfigFile }: FtrConfigProviderContext) => {
  const svlSearchConfig = await readConfigFile(require.resolve('../config.ts'));

  return {
    ...svlSearchConfig.getAll(),
    testFiles: [require.resolve('.')],
    junit: {
      reportName: 'Serverless Search Functional Tests - Lens',
    },
  };
};
```

Reply:
Thanks! I moved the files to `common/` and removed all nested `config.ts` in favor of the top level `common/config.ts`.